vividus-plugin-datetime/src/main/java/org/vividus/expression/DateExpression.java

```diff
@@ -58,7 +80,7 @@ public boolean hasCustomFormat()
 
     public String getCustomFormatString()
     {
-        return customFormatString;
+        return normalize(customFormatString);
```

Comment:
[getPeriod](https://github.com/vividus-framework/vividus/pull/3380/commits/7aa18209d7f81c7280b5cd44a443d89ff75e7ee4#diff-d1956a62e8bd9e3a84df60796619e587c6cac8724814db2df5344e602ce64441R110) loogicaly also should be normalized, for re-usability purpose, but at the same time according to the current implementation that does not make sense as input parameter for Period and Duration by it is meaning should not had `\\,` so that does not make sense to do, agree ?